app/assets/stylesheets/base/_buttons.scss

```diff
@@ -75,8 +75,8 @@ button[type="submit"],
 }
 
 .btn-small {
-  padding: ($small-spacing/2);
-  font-size: 12px;
+  padding: ($small-spacing/2) !important;
```

Comment:
Hi guys! :smile: 
@brunoocasali congratulations man, be awesome!!
@vyper I'll test it :+1: 

thanks for all!